docs/source/manifest_generation.rst

```diff
@@ -0,0 +1,177 @@
+Generate a manifest
+===================
+A **manifest** is a structured file containing metadata that adheres to a specific data model. This page covers different ways to generate a manifest.
+
+Prerequisites
+-------------
+
+**Before Using the Schematic CLI**
```

Comment:
You can probably refer to the installation guide instead of having all of this https://schematicpy.readthedocs.io/en/schematic-252_a/installation.html. 

There is a way to link to the rst documents so that it shows up as a link in the docs

Reply:
Just added the link there. Thanks!